### ISSUE TITLE
Docker column is empty

### ISSUE THREAD

**Issue by Saduff** [dalance/procs#236]
I'm running procs as root, but the docker column is empty. My config:
```toml
[[columns]]
kind = "Docker"
style = "BrightGreen|Green"
numeric_search = false
nonnumeric_search = true
align = "Left"

[docker]
path = "unix:///var/run/docker.sock"
```

I can access the docker socket using curl:
```sh
curl --unix-socket /var/run/docker.sock http://localhost/containers/json
```

**Comment by dalance** [comment]
Could you try https://github.com/dalance/procs/commit/fe3b992299656f0b7a30c0e926032f59f7fc2cf0 ?

**Comment by Saduff** [comment]
Still empty unfortunately. Seems to be because my cgroups don't start with `/docker`. Here's an example of my cgroups:
```
/system.slice/docker-9157725d9d022e72339c6e80f52d57ce354baa673e97c860d16d3df402af9f72.scope
/system.slice/docker.service
```

The processes with `/system.slice/docker.service` are either `docker-proxy` or `containerd-shim-runc-v2`.

**Comment by dalance** [comment]
Could you provide `cat /proc/[pid]/cgroup` of a process which belongs docker?

**Comment by Saduff** [comment]
```
# cat /proc/300709/cgroup                                                                     
0::/system.slice/docker-b508e2673897c4f521d858f56dab18e31ddbfc1f83300fb9793c8a4052af8f05.scope
```

However, in procs, there was no `0::` in the beginning. I checked by putting a `println!` of `cgroup_name` into procs.

**Comment by dalance** [comment]
OK. Could you try the following patch?

```diff
diff --git a/src/columns/docker.rs b/src/columns/docker.rs
index 910a97c..256bd87 100644
--- a/src/columns/docker.rs
+++ b/src/columns/docker.rs
@@ -104,6 +104,17 @@ impl Column for Docker {
                         ret = String::from("?");
                         break;
                     }
+                } else if cgroup_name.starts_with("/system.slice/docker-") {
+                    let container_id = cgroup_name
+                        .replace("/system.slice/docker-", "")
+                        .replace(".scope", "");
+                    if let Some(name) = self.containers.get(&container_id) {
+                        ret = name.to_string();
+                        break;
+                    } else {
+                        ret = String::from("?");
+                        break;
+                    }
                 }
             }
             ret
```

**Comment by Saduff** [comment]
It's working with the patch. Thanks!

**Comment by dalance** [comment]
Thanks!
I released v0.12.1.

**Comment by michabbb** [comment]
@dalance i am using kubuntu and i don´t see anything when calling `procs <container-name>`

procs version: `procs 0.13.0`
cat /proc/300510/cgroup: `0::/system.slice/docker.service`

my user is in the "docker" group

and the permissions of the socket file look okay to me:

```
Permissions Size User Group  Date Modified Name
srw-rw----     0 root docker 17 Jan 22:26  /var/run/docker.sock
```

but my output always is:

```
 PID:▲ User │ TTY CPU MEM CPU Time │ Command
            │     [%] [%]          │  
```

the config has the correct link to the socket-file:

```
[docker]
path = "unix:///var/run/docker.sock"
```

and calling `curl --unix-socket /var/run/docker.sock http://localhost/containers/json` works.

maybe you have an idea of where the problem is.

thanks !
